app/models/fact_check.rb

```diff
@@ -56,6 +57,11 @@ def self.get_exported_data(query, team)
     data
   end
 
+  def clean_tags
+    return if self.tags.blank?
+    self.tags = self.tags.map! { |tag| tag.strip.gsub(/^#/, '') }.uniq
```

Comment:
Not a blocker, but: `map!` already changes `self.tags` in place, so the `self.tags = self.tags.map!` is redundant (that's true for any method that ends on `!`)... but since at the end you use `uniq`, which doesn't change the attribute in place, I suggest you do:

```ruby
self.tags = self.tags.collect{ |tag| tag.strip.gsub(/^#/, '') }.uniq
```